core/src/main/java/org/frankframework/util/flow/MermaidFlowGenerator.java

```diff
@@ -67,6 +65,8 @@ public class MermaidFlowGenerator implements IFlowGenerator {
 
 	private static final String ADAPTER2MERMAID_XSLT = "/xml/xsl/adapter2mermaid.xsl";
 	private static final String CONFIGURATION2MERMAID_XSLT = "/xml/xsl/configuration2mermaid.xsl";
+	// List that contains all class patterns that extend FileSystemListener or FileSystemSender
```

Comment:
Yes, I agree and I understood that, while coding. It is a loose dependency on the class names and since we've concluded it is only used for coloring stuff, it is not that bad.